### PR TITLE
[Validator] add German translations for the Week constraint messages

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | 
| License       | MIT

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
please rebase :)

File: src/Symfony/Component/Validator/Resources/translations/validators.de.xlf [hunk 1]
```diff
@@ -452,19 +452,19 @@
             </trans-unit>
             <trans-unit id="116">
                 <source>This value does not represent a valid week in the ISO 8601 format.</source>
-                <target state="needs-translation">This value does not represent a valid week in the ISO 8601 format.</target>
+                <target>Dieser Wert ist keine Wochenangabe im ISO 8601-Format.</target>
             </trans-unit>
             <trans-unit id="117">
                 <source>The week "{{ value }}" is not a valid week.</source>
-                <target state="needs-translation">The week "{{ value }}" is not a valid week.</target>
+                <target>"{{ value }}" ist keine gültige Woche.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]: should be change the english source to `"{{ value }}" is not a valid week.`?

Comment (xabbuh) [reply src/Symfony/Component/Validator/Resources/translations/validators.de.xlf]: sounds reasonable: #57988

**Comment by nicolas-grekas** [comment]
Thank you @xabbuh.

**Pull request merged**